services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -3225,8 +3225,9 @@ public void setStoreCurrentVersion(String clusterName, String storeName, int ver
               "Unable to update store:" + storeName + " current version since store writeability is false");
         }
       }
+      int previousVersion = store.getCurrentVersion();
       store.setCurrentVersion(versionNumber);
-
+      realTimeTopicSwitcher.transmitVersionSwapMessage(store, previousVersion, versionNumber);
```

Comment:
If we let every region to send this Version Swap message to every RT, there might be N * N Version Swap message for N reigions. We might only need one region to be responsible for sending Version Swap to N region. Please correct me if I am wrong.

Reply:
In this first approach we're only doing the local RT.  This 'should' only write to the local colo.